.circleci/config.yml

```diff
@@ -4,37 +4,55 @@
 version: 2
 jobs:
   build:
+    working_directory: ~/java
+    parallelism: 1
     docker:
-      # specify the version you desire here
-      - image: circleci/openjdk:8-jdk
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
-
-    working_directory: ~/repo
+    - image: circleci/openjdk:8
 
     environment:
       # Customize the JVM maximum heap limit
       MAVEN_OPTS: -Xmx3200m
 
     steps:
+      - run: sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli --upgrade
```

Comment:
No need to do it all in one line:
https://github.com/Patreon/patreon_py/blob/master/.circleci/config.yml#L323

Reply:
For whatever reason I could not get that syntax to work (see build history), so I reverted to this way of doing things